kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -473,27 +473,46 @@ def get_vg_fencer_cmd(self, vg_uuid):
-                logger.warn(failure)
-                return self.inc_vg_failure_cnt(vg), failure
+        if r.get_renewal_last_result() == 1:
+            self.reset_vg_failure_cnt(vg)
+            return None
+
+        def is_heartbeat_timeout(timeout):
+            return abs(last_check - last_success) > timeout
+
+        try:
+            last_check = int(linux.monotime())
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)